pipeline.py

```
@@ -30,18 +30,30 @@
-    # Import our compatibility shim
-    import imghdr
+    # Create a minimal shim for compatibility
+    class imghdr:
+        @staticmethod
+        def what(file, h=None):
+            return None  # PaddleOCR will handle format detection
 
 # Direct execution: python pipeline.py
 import config
 from config import PADDLEOCR_LANG, OCR_LANG
 from config import *
```

Ruff (0.11.9)

`from config import *` used; unable to detect undefined names

(F403)